models/ToDoModel.ts

```diff
@@ -19,6 +20,10 @@ const ToDoSchema: Schema = new Schema({
     required: true,
     default: false,
   },
+  user: {
```

Comment:
why are we adding users with todos?
you should reference the todos to the userSchema